layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend

```diff
@@ -9,6 +9,14 @@ SRC_URI_append = " \
         file://0006-NFLX-2019-001-Resour-Consump-Low-MSS.patch \
         file://USB3-stability-fix.patch \
         file://0001-Add-support-for-NXP-PCA9956B-LED-controller.patch \
+        file://0001-Revert-MLK-17537-8-drm-mxsfb-Add-support-for-mode_va.patch \
+        file://0002-Revert-MLK-17537-9-gpu-imx-dcss-Add-support-for-mode.patch \
+        file://0003-Revert-MLK-17537-10-drm-imx-dcss-Add-support-for-mod.patch \
+        file://0004-Revert-MLK-17537-11-arch-arm64-fsl-imx8mq-evk-Reconf.patch \
+        file://0005-gpu-imx-dcss-Update-clocks.patch \
+        file://0006-drm-bridge-nwl-Improve-the-clock-calculation.patch \
+        file://0007-arch64-dts-fsl-imx8mq-Update-DCSS-LCDIF-and-DSI-cloc.patch \
+        file://0001-don-t-force-2-as-minimum-lanes-number.patch \
```

Comment:
This patch was created by me, and it removes the limitation to use at least 2 DSI lanes. The limitation of lanes >2 was added by NXP patches. For the RPi display we need to use 1 DSI lanes